src/cursor/cur_file.c

```diff
@@ -263,14 +263,19 @@ __wt_curfile_insert_check(WT_CURSOR *cursor)
 	WT_CURSOR_BTREE *cbt;
 	WT_DECL_RET;
 	WT_SESSION_IMPL *session;
+	int tret = 0;
```

Comment:
Our style is to separate declarations from initialization, so `tret = 0;` should be a separate line, just after `cbt = ...`.